services/brig/src/Brig/User/Auth.hs

```diff
@@ -81,11 +82,14 @@ sendLoginCode phone call force = do
 lookupLoginCode :: Phone -> AppIO (Maybe PendingLoginCode)
 lookupLoginCode phone = Data.lookupKey (userPhoneKey phone) >>= \case
     Nothing -> return Nothing
-    Just  u -> Data.lookupLoginCode u
+    Just  u -> do
+      Log.debug $ field "user" (toByteString u) . msg (val "User.lookupLoginCode")
+      Data.lookupLoginCode u
```

Comment:
Don't think this is needed, also covered on the `login` endpoint

Reply:
this one is only used in an internal endpoint `/i/users/login-code` so maybe we dont need it